plugins/rts54hub/rts54hub.quirk

```diff
@@ -21,3 +21,22 @@ FirmwareSize = 0x30000
 Rts54TargetAddr = 0x20
 Rts54I2cSpeed = 0x2
 Rts54RegisterAddrLen = 0x04
+Rts21xxFwUpdateMode = 1
```

Comment:
Is this always going to be a binary decision of 1 or 2?  If so, why not just having something like `Rts21xxBackgroundUpdate=False`